src/Scaffolding/VS.Web.CG.Mvc/Templates/Blazor/Create.cs

```diff
@@ -0,0 +1,458 @@
+// ------------------------------------------------------------------------------
```

Comment:
Formatting in this class appears off. Not sure if its github.com or the code, itself

Reply:
This is a file generated from T4 Processor, we can't control formatting in this file unfortunately since it'll get regenerated every build.